configurations/perf/regression-nosqlbench-gce.yaml

```diff
@@ -0,0 +1,8 @@
+test_duration: 360
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+gce_instance_type_db: 'e2-highmem-8'
```

Comment:
Are you going to use the default local_ssd_count?
(gce_n_local_ssd_disk_db: 3 is the default)

Reply:
I'm sticking with the default for now - might tweak this in a future PR